blog/_posts/2024-12-02-2nd-hackathon.md

```diff
@@ -0,0 +1,76 @@
+---
+ title: "A look back at the 2nd ROOT Hackathon: Python, Docs, Tutorials"
+ layout: archive
+ author: Monica Dessole
+---
+
+
+You may have heard about [the second ROOT Hackathon](https://indico.cern.ch/event/1463778/), that took place at IdeaSquare, CERN, last week! This in-person event was held on November 25-27. After a few days of recovering our energies, it's time to look back at this vibrant three days event, so let's go. 
+
+<center>
+    <img
+    src="{{'/assets/images/2nd-hackathon-intro.jpg' | relative_url}}"
+    alt="Hackathon opening."
+    style="width: 70%" width="1600" height="749" />
+</center>
+
+### The heart: participants
+
+This second hackathon vibrated with positive energy, fueled by a motivated group of 12 ROOT team members and 25 participants, many of which from CERN (thanks!). A standout achievement was our strides towards networking and team building, resulting in participation of people from across Europe (Italy, Czech republic, Spain...) and beyond (one participant from Qatar!), regardless of their gender, age, or background. The attendees could join three teams with different working topics, namely Python interfaces, Python Documentation and Tutorial modernisation, each guided by two coordinators from the ROOT team. This event indeed required a bit of organising and planning to put participants in condition to give their best. However, definitively worth the investment, we got a lot out of this event! 
+
+### The venue: IdeaSquare
+
+The hackathon took place at [IdeaSquare](https://ideasquare.cern), CERN's innovation space to foster cross-connectivity and to ideate new solutions. 
+IdeaSquare aligns with and supports CERN’s mission. Its specific focus is on offering a platform for early-stage collaboration with CERN scientists. IdeaSquare offers a fully equipped kitchen and a working space consisting of 700 square meters, split into five meeting rooms in glass containers and two meeting rooms in its famous double-decker London bus. This unique setting added an unparalleled charm to our coding marathon, infusing our work with inspiration and a sense of collaboration.
+
+### The core: projects
+
+The core of this hackathon lies in the projects themselves. For this edition, we identified three main topics: Python interface extention, Python documentation enhancement and Tutorial modernisation. Planning, ambitions and results are publicly visible and organised in a succinct [table on GH](https://github.com/orgs/root-project/projects/18).
```

Comment:
maybe you don't need to repeat the names of the three main topics here?

Reply:
Repetition has been removed after moving paragraph about venue